astroquery/mast/services.py

```diff
@@ -288,7 +288,27 @@ def service_request_async(self, service, params, page_size=None, page=None, use_
             catalogs_request.extend(self._build_catalogs_params(params))
         else:
             headers['Content-Type'] = 'application/json'
-            catalogs_request = params
+
+            # Parameter syntax needs to be updated only for PANSTARRS catalog queries
+            if service.lower() == 'panstarrs':
+                catalogs_request.extend(self._build_catalogs_params(params))
```

Comment:
This call updates the syntax of the parameters so it's readable in the backend. It needs to be made here so that users can add criteria decorators to their PANSTARRS queries, despite it being an API that requires params to be passed as JSON. It returns the params in list format, so I then have to convert it back to dictionary so that it can be dumped as a JSON dict later in the request.